artemis-commons/src/main/java/org/apache/activemq/artemis/api/core/ActiveMQLargeMessageInterruptedException.java

```diff
@@ -16,8 +16,6 @@
  */
 package org.apache.activemq.artemis.api.core;
 
-/**
- */
 // XXX
```

Comment:
Missed some extra "comment" that could also get deleted.

Reply:
Removed.